src/Serilog/Serilog.csproj

```diff
@@ -0,0 +1,74 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Simple .NET logging with fully-structured events</Description>
+    <VersionPrefix>2.4.1</VersionPrefix>
```

Comment:
I'm expecting no functional changes. Either way, we will get a new `-dev-0000x` build and can do further checking right?